ocm/__main__.py

```diff
@@ -234,6 +236,37 @@ def upload(parsed):
     )
 
 
+def add_cref(parsed):
+    raw = _parse_yaml_or_json(parsed.file)
+    component = raw['component']
+    component_references: list = component['componentReferences']
+
+    name = parsed.name or parsed.cname.split('/')[-1]
+    version = parsed.version
+    if not version:
+        parts = parsed.cname.split(':')
+        if len(parts) != 2:
+            print('error: unable to extract `:<version>`-suffix from component-name')
```

Comment:
nit: I think it is better to hint caller about the expected input, like so:
"expected `<component-name>:<version>`, but saw {actual-input}`